src/analysis/normed_space/star/multiplier.lean

```diff
@@ -60,14 +60,19 @@ open nnreal continuous_linear_map mul_opposite
 
 universes u v
 
+-- Backporting note: this used to extend `(A →L[𝕜] A) × (A →L[𝕜] A)`
+-- rather than having fields `fst` and `snd`.
+-- However the mathlib4 refactor https://github.com/leanprover-community/mathlib4/pull/3840
+-- when backported caused a timeout here. I don't understand why this timeout occurs,
```

Comment:
I can have a look at this tomorrow

Reply:
Thanks. I'm not sure that it's actually any worse this way. In fact, a benefit of the refactor I was forced to do here is that the fields could be called `L` and `R` as originally intended!